apps/web-portal-client/lib/index.jsx

```diff
@@ -1,13 +1,17 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import {HashRouter} from 'react-router-dom'
+import {Provider} from 'react-redux'
+import store from './store'
 import {AppContainer} from './app/AppContainer'
 import './index.css'
 
 // TODO: gh-23 Add Login page as component here
```

Comment:
looks like a leftover, that task was already closed a times ago

Plz, remove this one, I'm prepping sep issue to enable bot which would inform us about such kind of todos